site/content/en/docs/Reference/API/Kustomization File/configMapGenerator.md

```diff
@@ -6,3 +6,81 @@ weight: 6
 description: >
     Generate ConfigMap resources.
 ---
+
+## configMapGenerator
```

Comment:
I think it'll save you time to document the `configMapGenerator` field with the built-in `ConfigMapGenerator` transformer: https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_configmapgenerator_. `configMapGenerator` uses the built-in `ConfigMapGenerator` under the hood and they have essentially the same sub-fields. It'll be much easier for the documentation of one to reference the other instead of repeating ourselves. The built-in transformers are a bit confusing. LMK if you need help understanding.

Reply:
I added an `API/Generators` section. The ConfigMap and Secret generator types reference the common `GeneratorArgs` type. Please let me know if this is what you had in mind.